modules/lfstransfer/transfer/README.md

```diff
@@ -0,0 +1,8 @@
+# LFS SSH transfer protocol server
```

Comment:
Why not just reference but copy?

Reply:
It's ~1000 lines of code from a well-known & robust implementation of the protocol, under an MIT license. I was told in the issue discussion that integrating this external dependency would be fine, so it seemed the obvious choice to make.

Or if you actually meant to ask why I vendored it instead of just importing the module: see commit https://github.com/go-gitea/gitea/pull/31516/commits/8492908340902e8c788eb4f07609685359089f56, these changes were needed but couldn't have been made upstream. also the last tag on upstream was from ~2 years ago so we'd get an older version of this code without logging etc.

Reply:
regarding copyright headers: acknowledged, will add them in a bit

Reply:
@ConcurrentCrab I've cloned the upstream to https://gitea.com/gitea/git-lfs-transfer. Could you send a PR to it for the changes to the library instead (if they don't get merged upstream)?

We could also tag that repo on a more recent commit so that a `go get` will give us the expected version.

Thanks again for your work on this <3

Reply:
That's correct, the later of having a fork instead of vendoring it. Thanks for being so understanding :)

Reply:
@techknowlogick made a PR: https://gitea.com/gitea/git-lfs-transfer/pulls/1

Reply:
@techknowlogick any updates on this?